TestVectorsAwsCryptographicMaterialProviders/runtimes/python/fuzz_generator.py

```diff
@@ -150,7 +157,21 @@ def create_key_description(draw, keyring_type: str, test_type: str, kms_key: str
     else:
         raise ValueError(f"Unknown keyring type: {keyring_type}")
 
-# For the KMS keys
+
```

Comment:
extra line. As code grows this line creates a lot of noise. 1 line between two functions are great but more then 1 starts creating noise. 
```suggestion
```